<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UrsaPQ Remote</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainWidget, #statusBar {
	background-color: rgb(255, 255, 255);
}

QGroupBox {
    border: 1px solid gray;
    border-color: #003959;
    font-size: 14px;
    border-radius: 10px;
}

QGroupBox::title {
    subcontrol-position: top left;
 	border-top-left-radius: 10px;
 	border-bottom-right-radius: 10px;
    padding: 5px 20px;
    background-color: #003959;
    color: rgb(255, 255, 255);
}

* [objectName$=&quot;SL&quot;]{
    background-color: grey;
    border: 0px solid;
 	border-top-right-radius: 10px;
 	border-bottom-left-radius: 10px;
}

* [objectName$=&quot;MB&quot;]{
    border: 1px solid #003959;
    background-color: #006c96;
 	border-top-left-radius: 10px;
 	border-bottom-right-radius: 10px;
    color: rgb(255, 255, 255);
}

* [objectName$=&quot;MB&quot;]:hover{
    background-color: #01c6b2;
}

* [objectName$=&quot;MB&quot;]:pressed{
    background-color: #c6b301;
}
</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>376</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Vacuum</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="chamberPressure">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>45</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>75</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pre Vacuum pressure [mbar]:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cahmber pressure [mbar]:</string>
     </property>
    </widget>
    <widget class="QLabel" name="prevacPressure">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>75</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>105</y>
       <width>196</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pre Vacuum valves:</string>
     </property>
    </widget>
    <widget class="QLabel" name="prevacValves">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>105</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>135</y>
       <width>196</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Turbopump status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pumpStatus">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>135</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
    <widget class="QLabel" name="vacuum_SL">
     <property name="geometry">
      <rect>
       <x>346</x>
       <y>0</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="vacuum_MB">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>180</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Config</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>240</y>
      <width>376</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Detector</string>
    </property>
    <widget class="QLabel" name="detector_SL">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>0</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>15</y>
      <width>376</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Sample</string>
    </property>
    <widget class="QPushButton" name="sample_MB">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Config</string>
     </property>
    </widget>
    <widget class="QLabel" name="sample_SL">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>0</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>136</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Oven voltage</string>
     </property>
    </widget>
    <widget class="QLabel" name="sampleTemp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>45</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>300</y>
      <width>376</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Frame</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
